nautobot/docs/configuration/required-settings.md

```diff
@@ -150,16 +150,6 @@ setup](https://github.com/Suor/django-cacheops#setup).
 !!! warning
     [`CACHEOPS_REDIS`](#cacheops_redis) and [`CACHEOPS_SENTINEL`](#cacheops_sentinel) are mutually exclusive and will result in an error if both are set.
 
----
-
-### Task Queuing
-
-Task queues are configured by defining them within the [`RQ_QUEUES`](#rq_queues) setting. 
-
-Nautobot's core functionality relies on several distinct queues and these represent the minimum required set of queues that must be defined. By default, these use identical connection settings as defined in [`CACHES`](#caches) (yes, that's confusing and we'll explain below).
-
-In most cases the default settings will be suitable for production use, but it is up to you to modify the task queues for your environment and know that other use cases such as utilizing specific plugins may require additional queues to be defined. 
-
 #### CACHES
 
 The [`django-redis`](https://github.com/jazzband/django-redis) Django plugin is used to enable Redis as a concurrent write lock for preventing race conditions when allocating IP address objects, and also to define centralized Redis connection settings that will be used by RQ. The `CACHES` setting is required to to simplify the configuration for defining queues. *It is not used for caching at this time.*
```

Comment:
By removing the `Task Queueing` heading, `CACHES` now falls under the `Caching` heading. Given that `CACHES` isn't used for caching (wheeeeeeeee) this might not be ideal from a document flow perspective. Might be that the least worst option is to keep `### Task Queueing` as a level-three heading that only contains `#### CACHES` as a subheading?

Reply:
Ehhh yeah, right the confusing state of using the `CACHES` setting but not using it for caching, at least not out of the box.